src/examples/Logic_fastaSearch.py

```diff
@@ -0,0 +1,225 @@
+
+import webbrowser
+
+
+class logic:
```

Comment:
please choose a more telling name ... https://pspdfkit.com/blog/2018/naming-classes-why-it-matters-how-to-do-it-well/